server/middlewares/checkUser.js

```diff
@@ -25,5 +26,25 @@ class checkUser {
       });
     }
   }
+
+  /**
+   * This method Checks if the User's email is already in use .
```

Comment:
Refactor this. The method does not check whether the user is already in use according to its name, **checkSignIn**.

Reply:
done

Reply:
Not yet. Change the method's name to match its functionality.

Reply:
I added `CheckAuth` is it good??

Reply:
changed to` checkAuth`

Reply:
You have changed nothing. Use a helper for verifying the token. Reach out to Chris to the current agreed implementation for token verification.

Reply:
Ok sure!!!

Reply:
Awesome!